wandb/sdk/launch/agent/agent.py

```diff
@@ -23,6 +23,10 @@
 
 AGENT_POLLING_INTERVAL = 10
 
+AGENT_POLLING = "POLLING"
+AGENT_RUNNING = "RUNNING"
+AGENT_KILLED = "KILLED"
```

Comment:
Should we put these in an enum?

Reply:
python enums have some weird behavior if you set them to strings iirc so that eg Agent.KILLED != "KILLED" bc the types aren't the same, if you have a good workaround for that we could do that but otherwise since we actually need the string here this seems easier